pyproject.toml

```diff
@@ -26,15 +26,21 @@ classifiers = [
 ]
 license = "Apache 2.0"
 
+packages = [
+    { include = "meltano" },
+]
```

Comment:
@WillDaSilva fyi - just switching the package name and updating the dir path didn't work when calling `poetry build` or trying to import the package - since it was still failing to resolve the path. This seems to work though:

```
>>> from meltano import edk
>>> edk.__version__
'0.1.0'
```

Does that jive with what you where expecting or did I miss a step?

Reply:
> didn't work when [...] trying to import the package

What do you mean? The import path is `meltano.edk`, and it looks like it worked for you in the code block you posted.

As for `poetry build`, it worked for me.

Reply:
Yea sorry, in your original comment you mentioned just having to update the package name, and the directory path. But I also had to explicitly specify:

```
packages = [
  { include = "meltano" }
]
```

With that inplace everything works fine. Without out it:

```
edk) ➜  edk git:(1-use-namespaced-package) ✗ poetry build
Building meltano.edk (0.1.0)

  ModuleOrPackageNotFound

  No file/folder found for package meltano.edk

  at ~/.pyenv/versions/3.8.12/envs/edk/lib/python3.8/site-packages/poetry/core/masonry/utils/module.py:63 in __init__
       59│                             "from": str(src.relative_to(self._path)),
       60│                         }
       61│                     ]
       62│                 else:
    →  63│                     raise ModuleOrPackageNotFound(
       64│                         "No file/folder found for package {}".format(name)
       65│                     )
       66│
       67│         for package in packages:
```

I was just confirming that I actually needed the packages entry and wasn't missing something else instead.

Reply:
Ah yeah, my bad. I had not previously built a namespace package with Poetry, just other build systems, so I was going off of what I saw in the docs. Including `{ include = "meltano" }` makes sense to me, and seems to work well.